### ISSUE TITLE
LFH patch has 3 varients

### ISSUE THREAD

**Issue by Luciogi** [NyanCatTW1/robloxWineBuildGuide#29]
Which should we use use?

Explanation of patches: https://github.com/Frogging-Family/community-patches/commit/dcd70f2fb9f535b937996bef658f45db7defefdb#diff-0e03c28f1ead77d14e6891b731218421079c1ddceb22a18485e24b036a704466
```
0002-LFH-MR1628.mypatch
0002-proton_LFH-staging-winesync.mypatch
0002-proton_LFH-staging.mypatch
```

**Comment by Luciogi** [comment]
```
0002-LFH-MR1628.mypatch
0002-proton_LFH-staging-winesync.mypatch
0002-proton_LFH-staging.mypatch
```

**Comment by NyanCatTW1** [comment]
0002-proton_LFH-staging.mypatch seems like the right one
But well, it's actually already in the patch: https://github.com/NyanCatTW1/robloxWineBuildGuide/blob/main/roblox-wine-staging-v2.5.patch#L36

**Comment by NyanCatTW1** [comment]
Closed in #31
